IFComp/lib/IFComp/Schema/Result/User.pm

```diff
@@ -52,8 +53,6 @@ __PACKAGE__->table("user");
   is_nullable: 0
   size: 128
 
-User's real name
```

Comment:
@jmacdotorg This is one of the comments I'm talking about, in the schema class.

In the database, they're stored in the information_schema database, in the COLUMNS table. If you're using Sequel Pro, you can see/edit the comments in the comments column when viewing the table structure.